src/domains/api/actions.js

```diff
@@ -3,7 +3,7 @@ import * as CONSTANTS from './constants'
 import axios from 'axios'
 
 /* global localStorage */
-/* global fetch */
+///* global fetch */
```

Comment:
Can remove one line from here something like 
```suggestion
//* global fetch */
```